phpBB/styles/prosilver/template/viewtopic_body.html

```diff
@@ -224,7 +224,7 @@ <h2 class="poll-title"><!-- EVENT viewtopic_body_poll_question_prepend -->{POLL_
 			<div id="post_content{postrow.POST_ID}"<!-- IF postrow.S_POST_HIDDEN --> style="display: none;"<!-- ENDIF -->>
 
 			<!-- EVENT viewtopic_body_post_subject_before -->
-			<h3 <!-- IF postrow.S_FIRST_ROW -->class="first"<!-- ENDIF -->><!-- IF postrow.POST_ICON_IMG --><img src="{T_ICONS_PATH}{postrow.POST_ICON_IMG}" width="{postrow.POST_ICON_IMG_WIDTH}" height="{postrow.POST_ICON_IMG_HEIGHT}" alt="{postrow.POST_ICON_IMG_ALT}" title="{postrow.POST_ICON_IMG_ALT}" /> <!-- ENDIF --><a href="{postrow.U_MINI_POST}">{postrow.POST_SUBJECT}</a></h3>
+		<h3 {% if postrow.S_FIRST_ROW %}class="first"{% endif %}>{% if postrow.POST_ICON_IMG %}<img src="{{ T_ICONS_PATH }}{{ postrow.POST_ICON_IMG }}" width="{{ postrow.POST_ICON_IMG_WIDTH }}" height="{{ postrow.POST_ICON_IMG_HEIGHT }}" alt="{{ postrow.POST_ICON_IMG_ALT }}" title="{{ postrow.POST_ICON_IMG_ALT }}"> {% endif %}<a {% if postrow.S_FIRST_ROW && postrow.S_FIRST_UNREAD %}class="first-unread"{% endif %} href="{{ postrow.U_MINI_POST }}">{{ postrow.POST_SUBJECT }}</a></h3>
```

Comment:
Is the `S_FIRST_ROW and S_FIRST_UNREAD` statement correct? Should the class only be applied if the first post of a page is unread? Moreover, this will lead to a double space if the statement is false.

Reply:
i guess not should only be first unread